apps/mosaico/src/Mosaico/Footer.purs

```diff
@@ -258,11 +258,17 @@ column children = DOM.div { className: "mosaico-footer__contact-info__column", c
 
 section :: String -> Array JSX -> JSX
 section title children =
-  DOM.p
-    { className: "mosaico-footer__contact-info__section"
-    , children:
-        [ DOM.b { children: [ DOM.text title, DOM.text " " ] }
-        , DOM.div { children: children }
+  DOM.div -- WTF: this div does nothing... except that it prevents some really weird
```

Comment:
The library has underscored versions of functions that only take a `children` parameter. This could use `DOM.div_` instead.